### PR TITLE
Some gardening work

### PR DESCRIPTION
**Pull request by segiddins**
Saw the repo linked in https://ashfurrow.com/blog/up-and-running-with-ruby/ and thought I'd play around with it 🚀

### CODE REVIEW

**Comment by ashfurrow** [comment]
Fabbo, thanks! I'll take a look in the morning.

File: lib/danger_plugin.rb [hunk 1]
```diff
@@ -11,34 +10,45 @@ module Danger
...
+    def rubocop(files_to_lint)
+      rubocop_results = files_to_lint.flat_map do |f|
+        JSON.parse(`bundle exec rubocop -f json #{f}`)['files']
       end
+      rubocop_results.select { |f| f['offenses'].count > 0 }
+    end
 
-      markdown message
+    def offenses_message(offending_files)
+      require 'terminal-table'
```
Comment (ashfurrow) [inline lib/danger_plugin.rb]: Does this just happen to work with Markdown tables? Didn't see anything specific in their readme.

File: spec/danger_plugin_spec.rb [hunk 2]
```diff
@@ -32,25 +32,30 @@ module Danger
...
           output = @rubocop.status_report[:markdowns].first
 
           expect(output).to_not be_empty
 
           # A title
-          expect(output).to include("Rubocop violations")
+          expect(output).to include('Rubocop violations')
           # A warning
-          expect(output).to include("ruby_file.rb | 13 | Don't do that!")
+          expect(output).to include("ruby_file.rb | 13   | Don't do that!")
```
Comment (ashfurrow) [inline spec/danger_plugin_spec.rb]: Ah I see, so this is probably a consequence of using terminal-table with "line" for a heading, cool.

Comment (segiddins) [inline spec/danger_plugin_spec.rb]: yup!

File: lib/danger_plugin.rb (hunk 1 above)
Comment (segiddins) [inline lib/danger_plugin.rb]: output

---

### Rubocop violations

+--------------+------+----------------+
| File         | Line | Reason         |
+--------------+------+----------------+
| ruby_file.rb | 13   | Don't do that! |
+--------------+------+----------------+

**Comment by ashfurrow** [comment]
Added a circle config on another branch and pushed to master manually. Thanks again!

**Pull request merged**